pages/pricing.module.scss

```diff
@@ -2,3 +2,11 @@
   font-size: 14px;
   text-align: left;
 }
+
+.featureCard {
+  min-height: 501px;
```

Comment:
This was intentional.  I checked on a few screen resolutions, this one ensures the cards are the same height.  Was getting mixed results at 468.